sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -1871,6 +1871,51 @@ SIREPO.app.directive('validatedString', function(panelState, validationService)
     };
 });
 
+SIREPO.app.directive('viewLogIframe', function() {
+    return {
+        restrict: 'A',
+        scope: {
+            loadingLog: '<',
+            logHtml: '<',
+            logPath: '<',
+            wrapperDownloadLog: '<'
```

Comment:
How about just downloadLog? This directive doesn't need to know that there will be a "wrapper" around it.

Reply:
Ah, I didn't see that. Is there a reason for that function? I think we could get rid of it, rename wrapperDownloadLog to downloadLog, and the code should work.

Reply:
4d8ecf81fb4e98997af2c3b6de70315095700cc9